gazelle_haskell_modules/rule_generation.go

```diff
@@ -244,7 +246,11 @@ func appendLabelMaps(a map[label.Label]bool, b map[label.Label]bool) {
 func handleRuleError(err error, r *rule.Rule, attr string) {
 	if err != nil {
 		fmt.Println("Error when analyzing target", r.Name())
-		fmt.Println(attr, "=", build.FormatString(r.Attr(attr)))
+		if expr := r.Attr(attr); expr != nil {
+			fmt.Println(attr, "=", build.FormatString(expr))
+		} else {
+			fmt.Printf("%s doesn't exist in rule %s\n", attr, r.Name())
+		}
```

Comment:
I changed this to check if the attribute exists. This would cause a null pointer deref in the case where we're using the new directive, because then there's no `srcs` attribute.